.changeset/soft-falcons-love.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-user-settings': patch
```

Comment:
This actually constitutes a breaking change to the API surface. You will want to set this to `minor`, and running `yarn build:api-reports` to get an updated api report to add to the pull request (see the failing build log)

Reply:
I changed it to minor, but I dont know how to handle the changeset description. 

AFAIK there is no "breaking" change that has to be fixed. I added the `ProfileInfoApi` into `apiRef: ApiRef<ProfileInfoApi & SessionApi>;`, but I have no idea whether the users have to pass something extra into the `apiRef` or whether anyone who creates a signin provider has to implement the `ProfileInfoApi`. So I am not sure what I should write into the description, and how to explain what needs to be changed.